module.exports = {
  conflictingReqError: {
    user: 'пользователь с таким email уже существует',
  },
  notFoundError: {
    general: 'запрашиваемый ресурс не найден',
    user: 'нет пользователя с таким id',
  },
  forbiddenError: {
    user: 'других пользователей удалять нельзя',
  },
  disconnectedError: 'нет соединения с базой данных',
};
